doc/CHEATSHEET.md

```diff
@@ -2065,9 +2065,8 @@ dummyInformer.addEventHandler(new ResourceEventHandler<Dummy>() {
 - Create namespaced `SharedIndexInformer` (informers specific to a particular `Namespace`):
 ```java
 SharedInformerFactory sharedInformerFactory = client.informers();
-SharedIndexInformer<Pod> podInformer = sharedInformerFactory.sharedIndexInformerFor(
+SharedIndexInformer<Pod> podInformer = sharedInformerFactory.inNamespace("default").sharedIndexInformerFor(
```

Comment:
Feels better, indeed but then all methods taking an `OperationContext` should be removed, right?

Reply:
I have provided utility methods like `inNamespace()`, `withName()` for configuring namespace/name. A user might also want to provide more filtering options like labels, fieldSelectors which can be done by these `OperationContext` methods.

Shall I introduce `withLabels()`, `withFields` methods in SharedInformerFactory or shall I keep these OperationContext methods?

Reply:
I'd rather have a consistent API i.e. either everything with `with` methods or with `OperationContext`, not a mix of both.